aiida/tools/importexport/dbexport/__init__.py

```diff
@@ -725,83 +739,81 @@ def _get_starting_node_ids(entities: List[Any], silent: bool) -> Tuple[DefaultDi
     :raises exceptions.ArchiveExportError:
     :return: entities_starting_set, given_node_entry_ids
     """
-    # Instantiate progress bar - go through list of `entities`
-    pbar_total = len(entities) + 1 if entities else 1
-    progress_bar = get_progress_bar(total=pbar_total, leave=False, disable=silent)
-    progress_bar.set_description_str('Collecting chosen entities', refresh=False)
-
-    entities_starting_set = defaultdict(set)
-    given_node_entry_ids = set()
+    entities_starting_set: DefaultDict[str, Set[str]] = defaultdict(set)
+    given_node_entry_ids: Set[int] = set()
+
+    # store a list of the actual dbnodes
+    total = len(entities) + (1 if GROUP_ENTITY_NAME in entities_starting_set else 0)
+    if not total:
+        return entities_starting_set, given_node_entry_ids
+
+    with progress_context(desc='Collecting chosen entities', total=total) as progress_bar:
+        for entry in entities:
+
+            # This returns the class name (as in imports). E.g. for a model node:
+            # aiida.backends.djsite.db.models.DbNode
+            # entry_class_string = get_class_string(entry)
+            # Now a load the backend-independent name into entry_entity_name, e.g. Node!
+            # entry_entity_name = schema_to_entity_names(entry_class_string)
```

Comment:
I guess we can actually remove this (?)
At least it does not really seem to make sense any longer in this context

Reply:
removed